assets/sass/local.sass

```diff
@@ -344,3 +344,31 @@ dt.summary-list__key__wider
 
 .text-align-center
   text-align: center !important
+
+.margin-top-bottom-0
+  margin-top: 0
+  margin-bottom: 0
+
+.print-link 
+    display: inline-block
+    margin: 0 0 15px -10px
+    position: relative
+    top: 10px
+    padding: 0.5em 0.5em 0.5em 38px
+    background: url(/assets/images/Printer_icon.png) no-repeat 5px 50%
+    background-size: 22px 24px
+
+.print
+  display: none !important
+
+@media print
+  .no-print, .govuk-header, .govuk-footer
```

Comment:
is this going to apply to every page and so hide the header and footers from all print screens?
If so, Is it a problem?

Reply:
Yes, it will apply to any printed pages. I did consider whether there would be a reason where we would show the on-screen header and footer when printing something but I don't think we will need to. But I'll check with designers whether printing the header/footers will ever be needed